app/gateway/2.8.x/install-and-run/helm-quickstart-enterprise.md

```diff
@@ -0,0 +1,323 @@
+
+1. Add the Jetstack Cert Manager Helm repository:
+
+        helm repo add jetstack https://charts.jetstack.io ; helm repo update
+
+2. Install Cert Manager:
+        
+        helm upgrade --install cert-manager jetstack/cert-manager \
+            --set installCRDs=true --namespace cert-manager --create-namespace
+
+## Configure Kong Gateway
+
+Configuring Kong Gateway requires a Namespace and configuration secrets. Our secrets contain Kong's enterprise license, admin password, session configurations, and Postgres connection details. If you do not have a `license.json` file, please contact your account manager.
+
+2. Create Kong Namespace for {{site.base_gateway}}:
+
+        kubectl create namespace kong --dry-run=client -oyaml | kubectl apply -f -
+
+3. Create Kong Enterprise License Secret:
+
+        kubectl create secret generic kong-enterprise-license --from-file=license=license.json -n kong --dry-run=client -oyaml | kubectl apply -f -    
+      
+      >These instructions must be run in the directory that contains your `license.json` file. 
+
+4. Create Kong config & credential variables:
+
+        kubectl create secret generic kong-config-secret -n kong \
+            --from-literal=kong_admin_password=kong \
+            --from-literal=portal_session_conf='{"storage":"kong","secret":"CHANGEME-secret-salt","cookie_name":"portal_session","cookie_samesite":"off","cookie_secure":false}' \
+            --from-literal=admin_gui_session_conf='{"storage":"kong","secret":"CHANGEME-secret-salt","cookie_name":"admin_session","cookie_samesite":"off","cookie_secure":false}' \
+            --from-literal=pg_host="enterprise-postgresql.kong.svc.cluster.local" \
+            --from-literal=pg_port="5432" \
+            --from-literal=password=kong \
+            --dry-run=client -oyaml \
+          | kubectl apply -f -
+
+## Deploy Kong Gateway
+
+{:.important}
+> The following 4 steps are temporary development steps and will be removed from the guide.
+> These steps are required to access the helm-chart before it is merged into production.
+> These steps require the [Github CLI](https://cli.github.com/).
+
+1. `gh repo clone Kong/charts ~/kong-charts-helm-project` 
+2. `cd ~/kong-charts-helm-project/charts/kong`
+3. `gh pr checkout 592`
+4. `helm dependencies update`
+
+Once all dependencies are installed and ready, deploy Kong Gateway to your cluster:
+
+1. Add the Kong Helm Repo:
+
+        helm repo add kong https://charts.konghq.com ; helm repo update
+
+2. Install Kong:
+
+        helm upgrade --install quickstart --namespace kong \
+          --values ./example-values/quickstart-enterprise-licensed-aio.yaml \
+          ./
+
+3. Wait for all pods to be in the `Running` state:
+
+        kubectl get po --namespace kong
+
+4. Once all pods are running, open Kong Manager in your browser at [https://kong.7f000001.nip.io](https://kong.7f000001.nip.io). 
+
+5. Log in with the Super Admin username and password combination: `kong_admin`:`kong`
```

Comment:
Is it worth adding a line here about "and once logged in you can enable the developer portal at <url>"?

Reply:
I believe the quick start scope leans towards being cluttered with distractions. If the mission is to have kong running on someone's local environment then we should limit our narrative to that exclusively.

If we want to have a section for further reading that links to other documents and maybe video resources I think that would probably be good to include!